client/components/telecom/telephony/scheduler/actions/export/telephony-scheduler-export.html

```diff
@@ -1,21 +1,21 @@
-<form class="ovh-form-flat telecom-scheduler-export"
+<form class="telecom-scheduler-export"
       name="schdulerExportForm"
       data-ng-submit="SchedulerExportCtrl.startExport()">
 
     <div class="modal-header">
-        <h3 class="modal-title pull-left"
+        <h3 class="modal-title float-left"
```

Comment:
`<h3>` element must be in your `modal-body` and you can get rid off `modal-title float-left`.